packages/core/src/utilities.ts

```diff
@@ -1,5 +1,13 @@
 import { StyleSheet, StyleProp, TextStyle } from "react-native";
-import { isString, isNumber, pick, pickBy, identity } from "lodash";
+import {
+  isString,
+  isNumber,
+  pick,
+  pickBy,
+  identity,
+  omitBy,
```

Comment:
In the interest of getting rid of things that might be causing the import errors, do we need these new ones?

Reply:
these are just lodash methods. 
Purpose of this is to remove properties which are not defined. 

Do you think this can be causing problem?